### PR TITLE
[ONNX] Extend ONNX frontend with SoftmaxCrossEntropyLoss function #20547

### LINKED ISSUE #20547: [Good First Issue]: Extend ONNX Frontend with Function SoftmaxCrossEntropyLoss
### Context

Neural networks are graphs consisting of nodes called operators. Each operator corresponds to a mathematical function, usually described in framework's documentation or an AI standard, such as ONNX.
OpenVINO ONNX Frontend is a component responsible for working with ONNX graphs and requires implementation of different ONNX operators in order to use ONNX models.
This task requires extending OpenVINO ONNX Frontend with Function SoftmaxCrossEntropyLoss.													
Necessary help will be provided by ONNX Fronted team.

### What needs to be done?

Operator details can be found in [ONNX Operators](https://github.com/onnx/onnx/blob/main/docs/Operators.md#SoftmaxCrossEntropyLoss)
More details can be found in [ONNX Changelog](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#SoftmaxCrossEntropyLoss-12)
1. Create `.hpp` and `.cpp` files for *Windows [here](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/frontend/src/op)													
2. Prepare an implementation of this operator in form of a function. SoftmaxCrossEntropyLoss-12 should be placed in [opset 1 namespace](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/unique.hpp#L13). SoftmaxCrossEntropyLoss-13 should be placed in [opset 13 namespace](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/softmax.cpp#L50).
3. Register the function in [ops_bridge.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/ops_bridge.cpp#L554-L560) while keeping alphabetical order
4. Create test model(s) in [ONNX models directory](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/tests/models/). OpenVINO test infrastructure then converts prototxt files to ONNX models - you will use those models later in tests
5. Add tests covering all use cases [here](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/onnx_import.in.cpp)
6. Check [Python xfailed tests](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/tests_python/test_backend.py) to find a test marked as a xfailed for added functionality. If any exist - remove corresponding lines and try to verify by using cmdline "python -m pytest -k name_of_test".
More details in [adding operators to ONNX Frontend guide](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/how_to_add_op.md)

### Example Pull Requests

- https://github.com/openvinotoolkit/openvino/pull/15076
- https://github.com/openvinotoolkit/openvino/pull/14936
- https://github.com/openvinotoolkit/openvino/pull/12510
- https://github.com/openvinotoolkit/openvino/pull/19428

### Resources

- [What is OpenVINO?](https://github.com/openvinotoolkit/openvino#what-is-openvino-toolkit)
- [Contribution guide](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [Blog post on contributing to OpenVINO](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [User documentation](https://docs.openvino.ai/)
- Guides on [ONNX Frontend architecture](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/architecture.md) and [ONNX Frontend tests](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/tests.md) are also available
- You can also see a [blog post on contributing to OpenVINO](https://medium.com/openvino-toolkit/how-to-contribute-to-an-ai-open-source-project-c741f48e009e)

### Contact points

@gkrivor

### Ticket

_No response_

### PR DESCRIPTION
**Pull request by tanishka321**
### Details:
 Extend ONNX frontend with SoftmaxCrossEntropyLoss function

### Tickets:
 Closes #20547

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
Hi @tanishka321 
Please, resolve merge conflicts

**Comment by tanishka321** [comment]
> Hi @tanishka321 Please, resolve merge conflicts

**Comment by tanishka321** [comment]
> 

yes, accidentally closed the pull request but I am still working on it. Will be updating it soon

**Comment by ilya-lavrenov** [comment]
@tanishka321 could you please resolve merge conflicts?
